core/src/main/java/io/aiven/klaw/dao/EnvMapping.java

```diff
@@ -0,0 +1,42 @@
+package io.aiven.klaw.dao;
+
+import io.aiven.klaw.converter.EnvTagListConverter;
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.IdClass;
+import jakarta.persistence.Table;
+import java.io.Serializable;
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@ToString
+@Getter
+@Setter
+@Entity
+@IdClass(EnvID.class)
+@Table(name = "kwenvmapping")
```

Comment:
is there a way without the need of this table ? thinking if this table is of much use